tests/e2e/tests/020_packages/020_publish.ts

```diff
@@ -0,0 +1,15 @@
+import { npmPublish, npmLogin } from '../../utils/process';
+
+export default function() {
+  return new Promise((resolve, reject) => {
+    npmLogin('admin', 'blabla', 'foo@bar.com')
```

Comment:
I would like to warn you here that if you run this single test, this will not work.
I think you need to make sure about [that](https://github.com/Izak88/morose/blob/554fd3f70dc6634d8d585478068f1393634a6d12/tests/e2e/tests/010_user/010_login.ts#L13-L17).